cmd/tink-worker/main.go

```diff
@@ -16,41 +17,42 @@ func main() {
 	// parse and validate command-line flags and required env vars
 	flagEnvSettings, err := cmd.CollectFlagEnvSettings(os.Args[1:])
 	if err != nil {
-		fmt.Println("Error:", err)
+		fmt.Fprintln(os.Stderr, "Error:", err)
 		os.Exit(1)
 	}
 
 	// Here we retry failed connections to tink-server using a randomized interval between attempts.
 	// tink-worker is a daemon process, so we do not exit here.
 	var conn *grpc.ClientConn
 	for {
+		fmt.Fprintf(os.Stderr, "Obtaining tink server creds from %s...\n", flagEnvSettings.TinkServerURL)
 		creds, err := tink.ObtainServerCreds(flagEnvSettings.TinkServerURL)
 		if err == nil {
 			conn, err = tink.EstablishServerConnection(flagEnvSettings.TinkServerGRPCAuthority, creds)
 			if err == nil {
 				break
 			}
-			fmt.Printf("Error establishing gPRC connection to tink-server at %s: %v:", flagEnvSettings.TinkServerGRPCAuthority, err)
+			fmt.Fprintf(os.Stderr, "Error establishing gPRC connection to tink-server at %s: %v:", flagEnvSettings.TinkServerGRPCAuthority, err)
```

Comment:
I think there is a trailing `:` character, and the string also needs a `\n`.

Reply:
Fixed.